src/main/asciidoc/http.adoc

```diff
@@ -1127,7 +1127,7 @@ vertx.deployVerticle(() -> new AbstractVerticle() {
 }, new DeploymentOptions());
 ----
 
-When you interacting with the client possibly outside a verticle then you  you can safely perform
+When you interacting with the client possibly outside a verticle then you can safely perform
```

Comment:
I think there is another typo and it likely should be `When you are interacting`

Reply:
Done